.github/workflows/publish-pypi.yaml

```diff
@@ -0,0 +1,96 @@
+name: Publish to PyPI
+
+on:
+  push:
+    # Pattern matched against refs/tags
+    tags:
+      - '**'
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.x"
```

Comment:
@bennyrowland The build job is using the newest python 3 version. Let me know if you think it is best to pin the version to 3.8 (same version as test job in `ci-workflows.yaml`)

Reply:
I think this is probably fine, if it passes the tests under 3.8 then it doesn't really matter what is used to build it. And we probably don't need to continue to support 3.8 if we don't want to.